doc/nrf/device_guides/working_with_nrf/nrf52/developing.rst

```diff
@@ -82,6 +82,45 @@ If you want to add SMP Service to advertising data, refer to the :ref:`zephyr:sm
 
 .. fota_upgrades_over_ble_additional_information_end
 
+FOTA updates are also supported when MCUboot is in Direct-XIP mode.
+In this mode, the MCUboot bootloader boots an image directly from a given slot, so the swap operation is not needed.
+It can be used either with or without the revert mechanism support.
+For more information about Direct-XIP mode and the revert mechanism support, go to the :doc:`mcuboot:index-ncs` page.
+
+To use MCUboot in Direct-XIP mode together with FOTA updates, do the following:
+
+* Enable the :kconfig:option:`CONFIG_BOOT_DIRECT_XIP` Kconfig option in the MCUboot bootloader configuration.
+* Enable the :kconfig:option:`CONFIG_BOOT_BUILD_DIRECT_XIP_VARIANT` Kconfig option to build application update images for both slots.
+  For more information, go to :ref:`app_build_mcuboot_output` page.
+
+   * This Kconfig option automatically enables the :kconfig:option:`CONFIG_MCUBOOT_BOOTLOADER_MODE_DIRECT_XIP` Kconfig option to inform the application about the chosen MCUboot mode of operation.
+     Thus it is not required to set it manually if the Direct-XIP mode without the revert mechanism support is used.
+
+* Optionally, if you want to enable the revert mechanism support, you also need to:
+
+   * Enable the :kconfig:option:`CONFIG_BOOT_DIRECT_XIP_REVERT` Kconfig option in the MCUboot bootloader configuration.
+   * Enable the :kconfig:option:`CONFIG_MCUBOOT_BOOTLOADER_MODE_DIRECT_XIP_WITH_REVERT` Kconfig option to inform the application about the chosen MCUboot mode of operation.
```

Comment:
Do I understand correctly that the MCUmgr automatically confirms the image in the Direct-XIP mode like in the case of the standard (Swap) modes?

I just wanted to make sure that there is no need for the user to explicitly call the confirm API as it would require additional documentation.